internal/appconfig/machines.go

```diff
@@ -49,6 +49,36 @@ func (c *Config) ToReleaseMachineConfig() (*api.MachineConfig, error) {
 	return mConfig, nil
 }
 
+func (c *Config) ToConsoleMachineConfig() (*api.MachineConfig, error) {
+	mConfig := &api.MachineConfig{
+		Init: api.MachineInit{
+			// TODO: this is an ugly hack, abusing Hallpass as a
+			// stand-in for `/bin/sleep infinity`, simply because it
+			// should always be available.
+			Cmd:        []string{"dummy", "to", "keep", "machine", "running"}, // just to clear the CMD; Hallpass ignores it
+			Entrypoint: []string{"/.fly/hallpass"},
```

Comment:
not a fan but you can use `Exec` instead of `Entrypoint`+`Cmd`.  "Exec" will replace both at the same time. See https://flyio.discourse.team/t/apps-v2-and-experimental-config-features/2000/8

Reply:
Thanks for the tip! Unfortunately it doesn't seem to be working for me, which sounds like a bug somewhere in flyd.

In any case I'd really like to replace this hack with something sane. I'll probably bring this up again in Discourse today.